ui/src/routes/app/[id]/+page.svelte

```diff
@@ -177,19 +186,36 @@
 		<table class="table caption-bottom">
 			<thead class="">
 				<tr>
-					<th class="px-6 py-3 text-left text-xs font-medium tracking-wider uppercase">Title</th>
-					<th class="px-6 py-3 text-left text-xs font-medium tracking-wider uppercase">Author</th>
-					<th class="px-6 py-3 text-left text-xs font-medium tracking-wider uppercase"
+					{#if showImage.v === true}
+						<th class="px-6 py-3 text-left text-xs font-medium uppercase tracking-wider">Image</th>
+					{/if}
+					<th class="px-6 py-3 text-left text-xs font-medium uppercase tracking-wider">Title</th>
+					<th class="px-6 py-3 text-left text-xs font-medium uppercase tracking-wider">Author</th>
+					<th class="px-6 py-3 text-left text-xs font-medium uppercase tracking-wider"
 						>Last Updated
 					</th>
-					<th class="px-6 py-3 text-left text-xs font-medium tracking-wider uppercase"
+					<th class="px-6 py-3 text-left text-xs font-medium uppercase tracking-wider"
 						>Description
 					</th>
 				</tr>
 			</thead>
 			<tbody class="[&>tr]:hover:preset-tonal-primary divide-y divide-gray-200">
 				{#each slicedSource(data) as item (item.id)}
 					<tr class="hover:bg-gray-50">
+						{#if showImage.v === true}
+							<td class="w-52 p-0">
+								<a href="/item/{item.id}" target="_self" rel="noopener noreferrer">
+									<img
+										class="aspect-video object-cover lg:h-32 lg:min-w-48"
+										class:hue-rotate-90={!item.preview_url}
+										class:grayscale={!item.preview_url}
```

Comment:
So, one thing I do on the items page for the new relationship cards, is, have the greyscale removed on hover, rather than when we use the fallback image, this should be kept consistent with that functionality. You may also choose to remove the effect entirely or suggest something else.

https://github.com/disconsented/workshop-walker/blob/master/ui/src/routes/item/%5Bitem%5D/%2Bpage.svelte#L152